website_rate_product/models/mail_message.py

```diff
@@ -45,16 +45,19 @@ class ProductTemplate(osv.Model):
     _inherit = 'product.template'
 
     def _get_rating(self, cr, uid, ids, field_name, arg, context):
+        """
+        This method gets the rating for each rated template based on the
+        comments, the rating per comment is stored in the mail.message model
+        """
         res = {}
```

Comment:
You will need initialize the dict with all ids to avoid a `KeyError`
Check follow [example] (https://github.com/odoo/odoo/blob/51641d2634e48d07a3279acee405469e5d194e34/addons/product/product.py#L276)
`res = dict.fromkeys(ids, 0)`